tests/unit/Types/ContextFactoryTest.php

```diff
@@ -98,7 +98,7 @@ public function testReadsAliasesFromProvidedNamespaceAndContent()
          */
         public function testTraitUseIsNotDetectedAsNamespaceUse()
         {
-            $php = '<?php
+            $php = '<?php declare(strict_types=1);
```

Comment:
Think this change was unintended?

Reply:
Intended, this is consistent (tested) with the rest of the code.